packages/ui/src/utils/utils.ts

```diff
@@ -1,3 +1,6 @@
+import React, { ReactNode } from 'react'
```

Comment:
Importing `React` like this causes an ESLint warning since `react` doesn't have a default export. To use `React.X`, you need to import react as `import * as React from 'react'`, or we need to turn on synthetic default imports in the TS config. Best option is to just import whatever you need directly and drop the `React.` part.